spec/javascripts/components/intervention-spec.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-env jasmine, jquery */
+/* global GOVUK */
+
+describe('Intervention component', function () {
+  var FIXTURE
+
+  function loadInterventionComponent () {
+    var element = document.querySelector('[data-module="gem-intervention"]')
+    new GOVUK.Modules.Intervention(element).init()
+  }
+
+  beforeEach(function () {
+    spyOn(GOVUK.analytics, 'trackEvent')
+    spyOn(GOVUK.Modules, 'GemTrackClick').and.callFake(function () { this.start = function () {} })
```

Comment:
Is this line needed? This spy doesn't appear to be referenced again, and all we need to test is that the analytics `trackEvent` function was called - it doesn't matter if it doesn't do anything or not.